pages/getting-started/developers.md

```diff
@@ -140,6 +140,24 @@ You can also email us directly at [uswebdesignstandards@gsa.gov](mailto:uswebdes
 **For more information, visit:
 [https://pages.18f.gov/frontend/css-coding-styleguide/](https://pages.18f.gov/frontend/css-coding-styleguide/)**
 
+## JS customization
+
+The JavaScript for the standards is separated into components in the same manner as the visual interface which is all initialized with event handlers when the DOM is ready. These components are accessible as CommonJS modules that can be required in other JavaScript files which then must be fuilt for the browser. The components are currently not accessible in the global browser scope, but can be easily extended to be included by requiring `components` and setting it to a global scope:
```

Comment:
I think we might also want to get rid of the word "easily"...  Aside from our content guild generally discouraging such words, I don't think it's actually that easy given that someone has to set up their own browserify/webpack build system to do this, which can be really frustrating and time-consuming, especially for folks who are unfamiliar with such tools to begin with.